test/Iden3Helpers.test.js

```diff
@@ -1,218 +1,188 @@
-/* global artifacts */
-/* global contract */
-/* global web3 */
-/* global assert */
-
 const ethUtil = require('ethereumjs-util');
-const mimcUnit = require("../node_modules/circomlib/src/mimc_gencontract.js");
-const HelpersTest = artifacts.require("../contracts/test/Iden3HelpersTest");
-const bs58 = require('bs58');
-const bigInt = require('big-integer');
+const mimcUnit = require("circomlib/src/mimc_gencontract.js");
```

Comment:
You can remove all mimc related code as it is not used.

Reply:
Which of the following we should keep then:
babyjubjub.test.js
eddsa-babyjubjub.test.js
Iden3Helpers.test.js

Reply:
Discussed and removed